### ISSUE TITLE
Publish to mepla?

### ISSUE THREAD

**Issue by 9at8** [MatthewFluet/company-mlton#1]
It would be easier to get the package if it's on melpa. I can contribute if anything needs to be done! :raised_hands:

**Comment by MatthewFluet** [comment]
Agreed, publishing to melpa would be good.  I'm just not exactly familiar with the process.  Pointers and or PRs welcome.

**Comment by ghost** [comment]
I am available to help with this @MatthewFluet. I'd also like to help in general, where possible. This project would be great for Standard ML editor tooling. Wdyt?

**Comment by MatthewFluet** [comment]
@brettgilio Sure, that would be great!  Like I said, I'm just not familiar with the process of preparing an emacs package.
